app/src/main/java/com/github/libretube/constants/PreferenceKeys.kt

```diff
@@ -130,7 +131,7 @@ object PreferenceKeys {
     const val LAST_WATCHED_FEED_TIME = "last_watched_feed_time"
     const val HIDE_WATCHED_FROM_FEED = "hide_watched_from_feed"
     const val SELECTED_FEED_FILTER = "filer_feed"
-    const val FEED_SORT_ORDER = "sort_oder_feed"
+    const val FEED_SORT_ORDER = "sort_order_feed"
```

Comment:
I'll undo this because this will cause all users to lose the previously saved sort order when updating. It's not nice to have, but we can live with that typo.